### DISCUSSION TITLE
English Translation of the Ant Design Basic Design Principles 设计基础简版

### DISCUSSION THREAD

**Discussion by joyceZ916** [ant-design/ant-design#46394] [Show and tell]
Hello!

I came across the Ant Design 设计基础简版 in the project wiki, and I found it to be a useful summary and framework. Currently, it only exists in Chinese. I have created an English translation and hope that it proves to be helpful. Since I can't fork the wiki page, I'm sharing the .md file with my translation.

Cheers!

[Ant-Design-Basic-Design-Principles-设计基础简版.md](https://github.com/ant-design/ant-design/files/13642159/Ant-Design-Basic-Design-Principles-.md)

**Comment by yoyo837** [comment]
Thanks, updated the wiki.
